ios/brave-ios/Sources/Shared/Extensions/URLFormatterExtension.swift

```diff
@@ -0,0 +1,31 @@
+// Copyright 2024 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+import BraveCore
+import SwiftUI
+import UIKit
+
+extension URLFormatter {
+  public static func createAttributedString(
```

Comment:
I think you're better off creating a custom `Text` view for this instead btw. something like:

```swift
struct URLElidedText: View {
  var content: String

  @Environment(\.font) private var font: Font

  private var paragraphStyle: NSParagraphStyle {
    ...
  }

  var body: some View {
    Text(AttributedString("\u{200E}\(string)", attributes: [.font: font, paragraphStyle: paragraphStyle]))
  }
}```

Reply:
Didn't address this ☝️

Reply:
I only saw this after. Just pushed a commit to address it.